src/RageInput.cpp

```diff
@@ -12,6 +12,7 @@
 RageInput* INPUTMAN = nullptr; // global and accessible from anywhere in our program
 
 Preference<RString> g_sInputDrivers( "InputDrivers", "" ); // "" == DEFAULT_INPUT_DRIVER_LIST
+Preference<RString> g_sInputLinuxJoysticks( "InputLinuxJoysticks", "" ); // "" == DEFAULT_LINUX_INPUT_JOYSTICK_LIST
```

Comment:
Can we rename all this to `InputDeviceOrder`. Even though the current implementation is only specific to linux, I can see it potentially being useful in the future for other OSes as well. And I think this naming is a bit more clear to it's function.

Reply:
Makes complete sense, thanks! I've renamed the variable in a new commit.

For a future PR, where would you think I could document the functionality? Would Docs/Userdocs/Preferences.ini.md be the right place?